Sources/CLI/RunCommand.swift

```diff
@@ -315,3 +356,96 @@ struct ProcessIO {
         }
     }
 }
+
+struct OSFile: Sendable {
```

Comment:
Note: I've just copied a smaller portion of this same type we have for vminitd here. I'll see if it makes sense to make that public over in CZ, and we can just use directly.